### PR TITLE
"GDevelop Forum" and "GDevelop Wiki" buttons

### PR DESCRIPTION
**Pull request by araujo921**

### CODE REVIEW

**Comment by 4ian** [comment]
Nice! Could you post a screenshot of the result? :)
If it's ok, there will be a bit of commented code to clean ;)

**Comment by araujo921** [comment]
Yes, I'll comment... :-)
see below result... ^_^

fireshot capture 2 - gdevelop - http___localhost_3000_
fireshot capture 1 - gdevelop - http___localhost_3000_

**Comment by araujo921** [comment]
The below image is without comments of line 17 and lines 32 through 38. what do you think .. ???
I do not know if it's good.

fireshot capture 3 - gdevelop - http___localhost_3000_

**Comment by 4ian** [comment]
That's a good start! 😃  I think that the buttons are taking a bit too much space on the screen and are more visible than Open/Create a new project buttons.
The first thing that should be visible, centered on the screen, are these two buttons because it is what a new user need: two big buttons to create a new project or open an existing one :)

Instead I would have put GDevelop forum and GDevelop Wiki next to About GDevelop, using `FlatButton` component 😄 
The website is not useful here I think as it's mainly a website to introduce GDevelop and let users download it. So let's just have GDevelop Forum and GDevelop Wiki next to "About GDevelop". You can add `primary` prop on them to make them blue and a bit more visible if you wish :)

**Comment by araujo921** [comment]
fireshot capture 5 - gdevelop - http___localhost_3000_

**Comment by araujo921** [comment]
Okay, I'll make the changes.... :-)

**Comment by 4ian** [comment]
What about something like this? 😃 

screen shot 2017-08-23 at 22 54 47

That should be pretty simple and we do not need to complicate the codebase with new buttons :) I'd like to keep Material UI buttons (`RaisedButton` and `FlatButton`) as long as possible because they look nice and we need to have an interface that is consistent ;)

**Comment by araujo921** [comment]
I understand ... I just created the buttons and links so I did not have to be copying the styles ... You'd think it's better not to have these components, right?

**Comment by 4ian** [comment]
Yes I think they are not useful if we do something very simple :)

**Comment by araujo921** [comment]
Okay, I've already removed it and I'll send a new pull request.

**Comment by araujo921** [comment]
fireshot capture 6 - gdevelop - http___localhost_3000_


**what is the next step?**

**Comment by araujo921** [comment]
fireshot capture 7 - gdevelop - http___localhost_3000_

**Comment by 4ian** [comment]
The buttons looks good :)
But I don't understand why the Open A Project/Create A New Project buttons are shrinked in your last screenshot? Let them as they were before :)

**Comment by 4ian** [comment]
I would like to merge the GDevelop Forum/GDevelop Wiki but you've included another commit which is related to errors. Can you just make this PR with buttons?

Let's keep the Error component for another PR ;)

File: newIDE/app/src/MainFrame/Toolbar.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default class MainFrameToolbar extends Component {
             <ToolbarIcon
               onClick={this.props.openProject}
               src="res/ribbon_default/open32.png"
+              title="Open Project"
```
Comment (4ian) [inline newIDE/app/src/MainFrame/Toolbar.js]: What is this?

File: newIDE/app/src/MainFrame/Editors/StartPage.js [hunk 2]
```diff
@@ -63,6 +64,7 @@ export default class StartPage extends BaseEditor {
                 fullWidth
                 onClick={this.props.onOpen}
               />
+              <Divider />
```
Comment (4ian) [inline newIDE/app/src/MainFrame/Editors/StartPage.js]: No need for a Divider I think :)

File: newIDE/app/src/MainFrame/Editors/StartPage.js [hunk 3]
```diff
@@ -81,4 +94,4 @@ export default class StartPage extends BaseEditor {
       </Column>
     );
   }
-}
+}
```
Comment (4ian) [inline newIDE/app/src/MainFrame/Editors/StartPage.js]: It's a good practice to always have a new line at the end of a file: See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline 😉 
You can configure your text editor to automatically enforce this.

File: newIDE/app/src/Utils/Error.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+// Define error component style.
+const styles = {
+    paper: {
+    }
+};
+/**
+ * Error component
+ */
+export default class Error extends React.Component {
```
Comment (4ian) [inline newIDE/app/src/Utils/Error.js]: That's a good start too, but let's keep this for another Pull Request :)
It's easier to work and merge small Pull Request ;)

File: newIDE/app/src/MainFrame/Toolbar.js (hunk 1 above)
Comment (araujo921) [inline newIDE/app/src/MainFrame/Toolbar.js]: an element attribute... in other word, a tooltip... :-)

File: newIDE/app/src/Utils/Error.js (hunk 4 above)
Comment (araujo921) [inline newIDE/app/src/Utils/Error.js]: Okay... :+1:

File: newIDE/app/src/MainFrame/Editors/StartPage.js (hunk 2 above)
Comment (araujo921) [inline newIDE/app/src/MainFrame/Editors/StartPage.js]: Okay... it also shrunk buttons... :-)

remove and make new commit...???

**Comment by 4ian** [comment]
Sweet, merging this now :) I'm "squashing" this in a single commit.

**Comment by 4ian** [comment]
Thanks for your work and for the PR 👍 😄
